stage2/01-sys-tweaks/files/rc.local

```diff
@@ -73,4 +73,12 @@ if [ ! -f /home/umbrel/statuses/service-configured ]; then
     fi
 fi
 
+# Display connection details on the TTY
```

Comment:
You could add a `clear` here.

Reply:
I'm not sure we should clear the TTY here since there could be useful log output on the screen that we don't want to overwrite.